.circleci/config.yml

```diff
@@ -0,0 +1,38 @@
+version: 2
+jobs:
+  dendrite:
+    docker:
+      - image: matrixdotorg/sytest-dendrite
+    working_directory: /src
+    steps:
+      - checkout
+
+      # Set up dendrite
+      - run: 
+          name: Build Dendrite
+          command: ./build.sh
+      - run: 
+          name: Copy custom dendrite config
+          command: cp .circleci/dendrite-config.yaml dendrite.yaml
+      - run: 
+          name: Generating self-signed server keys
+          command: |
+            test -f server.key || openssl req -x509 -newkey rsa:4096 -keyout server.key -out server.crt -days 3650 -nodes -subj /CN=localhost
+            test -f matrix_key.pem || ./bin/generate-keys -private-key matrix_key.pem
```

Comment:
Can't we just keep a static set in `.circleci/` so that we don't have to wait for them to be generated each run?